Subtester/Assets/Scripts/PurchasesListener.cs

```diff
@@ -70,7 +70,7 @@ private void CreatePurchasePackageButtons()
                 {
                     Debug.Log("Package " + package);
                     if (package == null) continue;
-                    var label = package.PackageType + " " + package.StoreProduct.PriceString;
+                    var label = package.PackageType + " " + package.StoreProduct.PriceString + "\n" + package.StoreProduct.SubscriptionPeriod;

```

Comment:
Hmm I guess it will be pretty self-explanatory but I wonder if we should add a label to the period, like `SubscriptionPeriod: P1Y`. Mostly for people not as familiar with this. This wouldn't apply to inapp's though, where adding that might make it more confusing... I'm ok leaving as is though

Reply:
I actually added this for test purposes only, and then I noticed there is a log at the top of the app so I might just remove this completely